test/gluttony/record/consumer_test.clj

```diff
@@ -176,7 +177,39 @@
                          :receive-limit 10
                          :consume-limit 0
                          :long-polling-duration 20
-                         :exceptional-poll-delay-ms 0
```

Comment:
This test case intend to "heartbeat is bigger than heartbeat-timeout". But, throw exception by `(pos? exceptional-poll-delay-ms)`